lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -169,7 +165,11 @@ def eci_value(payment_method, options)
       end
 
       def authorize_request(money, payment_method, options)
-        commit('authorize', build_authorization_request(money, payment_method, options), 'AUTHORISED', 'CAPTURED', options)
+        if options[:account_funding_transaction]
+          aft_request(money, payment_method, payment_details.merge(**options))
```

Comment:
Seeing an error in the tests now - I think we need to pass in just `options` as the third argument here, since the `.merge` has already been called in the middle of the `authorize` method before referencing this separate sub-method

Reply:
already fixed! saw the failing tests in the GHA. TY!